frontend/app/src/composables/history/filter-paginate.ts

```diff
@@ -76,7 +70,7 @@ export const useHistoryPaginationFilter = <T extends Object, U, V>(
       offset,
       orderByAttributes: sortBy?.length > 0 ? sortBy : ['timestamp'],
       ascending: sortBy?.length > 0 ? sortDesc.map(bool => !bool) : [false]
-    };
+    } as U;
```

Comment:
What is the error here? Is this because of the extra params?

Reply:
Type 'W & { limit: number; offset: number; orderByAttributes: (keyof T)[] | string[]; ascending: boolean[]; }' is not assignable to type 'U'.     'U' could be instantiated with an arbitrary type which could be unrelated to 'W & { limit: number; offset: number; orderByAttributes: (keyof T)[] | string[]; ascending: boolean[]; }'.

Reply:
Ok, can you add a TODO there to figure out later? No need to spend more time on it right now.

It's preferable to not have casting, but figuring out type issues is usually a time-sink.

Reply:
Okay, will add todo to it, thanks.